.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,57 @@
-# Light It Pull Request Template ⚡⚡⚡
+# ⚡ TaskTitle - [###](https://app.clickup.com/t/###) ⚡
 
-_Resolve_: [closes -#](https://app.clickup.com/t/ticket_id)
+## 💻 What type of change is this?
 
-## Description and Requires ⭐
+- [ ] 💎 Feature
+- [ ] 🐛 Bug Fix
+- [ ] 📝 Documentation Update
+- [ ] 🎨 Styling
+- [ ] 🧑‍💻 Code Refactor
+- [ ] 🔥 Performance Improvements
+- [ ] ✅ Test
+- [ ] 🤖 Build / CI
+
+## ⭐ Description
 
 <!--
-Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
 Example:
 -->
 
-Ex. I've added support for authentication to implement Key Result 2 of OKR1. It includes
-model, table, controller and test. For more background, see ticket #CU-834794.
+I've added support for authentication to implement Key Result 2 of OKR1. It includes model, table, controller and test.
+
+<!--
+ONLY ADD SECTION IF A NEW PACKAGE IS ADDED
+### Requires
+This pr requires the following packages to be installed:
+- `package1`
+- `package2`
+The packages are used for `reason1` and `reason2`.
+
+Because of this, you must run `npm i` before starting.
+-->
 
-- [ ] This change requires a database update or migration
-- [ ] This change requires a documentation update
+## 📷 Screenshots
+
+<!--
+Please include before AND after screenshots of the change.
+-->
 
-## Proof of Test (optional) 📷
+### Before
 
-Include Screenshots if is necessary.
+### After
 
-### How Has This Been Tested? ❓
+## 💬 Comments
 
 <!--
-Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+Please describe any known issues, bugs, or unintended consequences with this change. Also, please include any additional comments you feel are relevant to the reviewer.
 -->
 
-- [ ] Test A
-- [ ] Test B
+This pr is blocked by #1234.
+Im awaiting backend changes to be merged before I can complete this, etc.
+This
 
-## Checklist ✅
+## ✅ Checklist
 
-- [ ] My code follows the style guidelines of this project
-- [ ] I have performed a self-review of my own code
-- [ ] I have made corresponding changes to the documentation
-- [ ] My changes generate no new warnings
-- [ ] I have added tests that prove my fix is effective or that my feature works
-- [ ] New and existing unit tests pass locally with my changes
-- [ ] Any dependent changes have been merged and published in downstream modules
```

Comment:
Dont remove this, the boirlerpalate is fullstack

Reply:
[suggestion](https://github.com/Light-it-labs/laravel/pull/9/files/d460547116827a0a7486f499381301cb077cc0a2#r1391097101)